boards/arm/mimxrt1060_evk/doc/mimxrt1060_evk.rst

```diff
@@ -0,0 +1,194 @@
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1060_evk/mimxrt1060_evk_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1060 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B0_09 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_12 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_13 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| WAKEUP        | GPIO            | SW0                       |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1060 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 600 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1060 SoC has eight UARTs. One is configured for the console and the
+remaining are not used.
+
+Programming and Debugging
+*************************
+
+The MIMXRT1060-EVK includes the :ref:`nxp_opensda` serial and debug adapter
+built into the board to provide debugging, flash programming, and serial
+communication over USB.
+
+To use the Segger J-Link tools with OpenSDA, follow the instructions in the
+:ref:`nxp_opensda_jlink` page using the `Segger J-Link OpenSDA V2.1 Firmware`_.
+The Segger J-Link tools are the default for this board, therefore it is not
+necessary to set ``OPENSDA_FW=jlink`` explicitly when you invoke ``make
+debug``.
+
+With these mechanisms, applications for the ``mimxrt1060_evk`` board
+configuration can be built and debugged in the usual way (see
+:ref:`build_an_application` and :ref:`application_run` for more details).
+
+The pyOCD tools do not yet support this SoC.
+
+Flashing
+========
+
+The Segger J-Link firmware does not support command line flashing, therefore
+the usual ``flash`` build system target is not supported.
```

Comment:
Add after this sentence:

    Instead, see the `MIMXRT1060-EVK User Guide`_ for flashing instructions.

Reply:
> Add after this sentence:
> 
> ```
> Instead, see the `MIMXRT1060-EVK User Guide`_ for flashing instructions.
> ```

Thank you, just checked the online doc, that flash programming thing is described in another application note. I updated the rst file giving the link to that AN for reference.